src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -178,6 +186,9 @@ private void model_PropertyChanged(object sender, System.ComponentModel.Property
                 case "FontSize":
                     RaisePropertyChanged("FontSize");
```

Comment:
FSize is used in updatevalue method (mainly to get the old value on Undo). I can rename this to FontSize - no harm !